js/classes.js

```diff
@@ -699,13 +697,7 @@ FitsFileLoader.prototype.getIntensityData = function() {
  */
 FitsFileLoader.prototype.getVarianceData = function() {
     this.log.debug("Getting spectra variance");
-    var index = this.getHDUFromName("variance");
-    if (index == null) {
-        index = this.getHDUFromName("ivar");
-        if (index != null) {
-            this.inverseVariance = true;
```

Comment:
Do you reset this elsewhere? For example, DESI spectra pass through "ivar" being the inverse variance as opposed to say, OzDES spectra which give the regular variance.